esp-hal/src/lib.rs

```diff
@@ -235,6 +235,8 @@ pub mod i2c;
 pub mod peripheral;
 #[cfg(all(feature = "unstable", any(soc_has_hmac, soc_has_sha)))]
 mod reg_access;
+#[cfg(all(feature = "unstable", any(feature = "esp32", feature = "esp32c6")))]
```

Comment:
```suggestion
#[cfg(all(feature = "unstable", any(sd_host))]
```

you'll need to fix up the metadata, if this works for the c6 we're missing the sd_host driver entry

Reply:
Can I use the `soc_has_sd` instead? Part of the `sdio` module will also include host drivers in the future, e.g. for the ESP32-P4.

Also, where are these `cfg` selectors defined? I see `soc_has_hmac` used, but not where it is defined.

Reply:
it's defined in esp-metadata; each listed peripehral creates a soc_has_* symbol